packages/orchestration/src/examples/sendAnywhere.contract.js

```diff
@@ -54,6 +92,14 @@ export const start = async (zcf, privateArgs, baggage) => {
 
   /** @type {import('../orchestration-api.js').OrchestrationAccount<any>} */
   let contractAccount;
+  const cell = harden({
+    get contractAccount() {
```

Comment:
"cell" also seems like strange terminology.

I wonder about `contractState.account` instead of `cell.contractAccount`.